### PR TITLE
Metadata review - WFP-DOFSTK updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates to DOFSTK cal csvs per metadata review project

### CODE REVIEW

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20150130.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-0209,CC_frequency_offset,-797.5547,
-43-0209,CC_oxygen_signal_slope,0.0002954526,
+43-0209,CC_frequency_offset,-797.5547,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_0209_Calibration_2015-01-30.zip > SBE 43F O0209 SOCAdjusted.pdf and 0209.cal
+43-0209,CC_oxygen_signal_slope,3.1147e-004,this is the adjusted Soc (O2 signal slope) value
 43-0209,CC_residual_temperature_correction_factor_a,-0.004551888,
 43-0209,CC_residual_temperature_correction_factor_b,0.0001997635,
 43-0209,CC_residual_temperature_correction_factor_c,-2.736936e-06,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20150130.csv]: JML looks ok

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20170726.csv [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-43-0209,CC_frequency_offset,-809.3336,
-43-0209,CC_oxygen_signal_slope,0.000312673,
+43-0209,CC_frequency_offset,-809.3336,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_0209_Calibration_2017-07-26.zip > SBE 43F O0209 SOCAjusted.pdf and 0209.cal 
+43-0209,CC_oxygen_signal_slope,3.2509e-004,adjusted Soc (O2 signal slope) value
 43-0209,CC_residual_temperature_correction_factor_a,-0.004386895,
-43-0209,CC_residual_temperature_correction_factor_b,0.000194881,
-43-0209,CC_residual_temperature_correction_factor_c,-2.67E-06,
+43-0209,CC_residual_temperature_correction_factor_b,1.948812e-004,
+43-0209,CC_residual_temperature_correction_factor_c,-2.674480e-006,
 43-0209,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20170726.csv]: Looks awesome!!

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20180524.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-0209,CC_frequency_offset,-7.985563e+002,
-43-0209,CC_oxygen_signal_slope,3.212782e-004,
+43-0209,CC_frequency_offset,-7.985563e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_0209_Calibration_2018-05-24.zip > SBE 43F O0209 SOCAdjusted 24May18.pdf and SBE43F_0209.cal
+43-0209,CC_oxygen_signal_slope,3.4045e-004,adjusted Soc (O2 signal slope) value
 43-0209,CC_residual_temperature_correction_factor_a,-3.754653e-003,
 43-0209,CC_residual_temperature_correction_factor_b,1.919485e-004,
 43-0209,CC_residual_temperature_correction_factor_c,-2.816449e-006,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20180524.csv]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02397__20140805.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2397,CC_frequency_offset,-838.72,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2397_Calibration_2014-08-05.zip > 43F O2397 SOCAdjusted.pdf
+43-2397,CC_oxygen_signal_slope,2.7885e-004,adjusted Soc (O2 signal slope) value
+43-2397,CC_residual_temperature_correction_factor_a,-4.0479e-003,
+43-2397,CC_residual_temperature_correction_factor_b,2.1156e-004,
+43-2397,CC_residual_temperature_correction_factor_c,-2.6781e-006,
+43-2397,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-02397__20140805.csv]: these match pdf

**Comment by rtravis-whoi** [comment]
FYI - vendor provided a couple missing cal docs - csvs have been updated and added to pull request.

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20161221.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2724,CC_frequency_offset,-7.936951e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2724_Calibration_2016-12-21.zip > SBE 43F O2724 SOCAdjusted.pdf and 2724.cal 
+43-2724,CC_oxygen_signal_slope,2.8504e-004,adjusted Soc (O2 signal slope) value
+43-2724,CC_residual_temperature_correction_factor_a,-3.856074e-003,
+43-2724,CC_residual_temperature_correction_factor_b,1.632224e-004,
+43-2724,CC_residual_temperature_correction_factor_c,-2.184744e-006,
+43-2724,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20161221.csv]: ok

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20171223.csv [hunk 6]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2724,CC_frequency_offset,-7.840658e+002,
-43-2724,CC_oxygen_signal_slope,3.539816e-004,
+43-2724,CC_frequency_offset,-7.840658e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2724_Calibration_2017-12-23.zip > SBE 43F O2724 SOCAdjusted.pdf and 2724.cal 
+43-2724,CC_oxygen_signal_slope,3.5741e-004,adjusted Soc (O2 signal slope) value
 43-2724,CC_residual_temperature_correction_factor_a,-5.193974e-003,
 43-2724,CC_residual_temperature_correction_factor_b,1.740608e-004,
 43-2724,CC_residual_temperature_correction_factor_c,-2.259399e-006,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20171223.csv]: yup

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20190312.csv [hunk 7]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2724,CC_frequency_offset,-7.8859e+002,
-43-2724,CC_oxygen_signal_slope,2.8979e-004,
+43-2724,CC_frequency_offset,-7.885854e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2724_Calibration_2019-03-12.zip and 2724.cal
+43-2724,CC_oxygen_signal_slope,2.8979e-004,adjusted Soc (O2 signal slope) value
 43-2724,CC_residual_temperature_correction_factor_a,-5.071532e-003,
 43-2724,CC_residual_temperature_correction_factor_b,2.110971e-004,
 43-2724,CC_residual_temperature_correction_factor_c,-2.830452e-006,
```
Comment (johnmlund) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20190312.csv]: ok

Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20190312.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20150218.csv [hunk 8]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2725,CC_frequency_offset,-805.62,
-43-2725,CC_oxygen_signal_slope,0.00029059,
+43-2725,CC_frequency_offset,-8.056188e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2725_Calibration_2015-02-18.zip > SBE 43F O2725 SOCAdjusted.pdf and 2725.cal
+43-2725,CC_oxygen_signal_slope,0.00029059,adjusted Soc (O2 signal slope) value
 43-2725,CC_residual_temperature_correction_factor_a,-0.004035528,
 43-2725,CC_residual_temperature_correction_factor_b,0.0001810064,
 43-2725,CC_residual_temperature_correction_factor_c,-2.705844e-06,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20150218.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20151223.csv [hunk 9]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2725,CC_frequency_offset,-7.999255e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2725_Calibration_2015-12-23.zip > SBE 43F O2725 SOCAjusted.pdf and 2725.cal
+43-2725,CC_oxygen_signal_slope,2.9678e-004,adjusted Soc (O2 signal slope) value
+43-2725,CC_residual_temperature_correction_factor_a,-3.386928e-003,
+43-2725,CC_residual_temperature_correction_factor_b,1.610983e-004,
+43-2725,CC_residual_temperature_correction_factor_c,-2.513118e-006,
+43-2725,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20151223.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20161124.csv [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
-43-2725,CC_residual_temperature_correction_factor_a,-2.38E-03,
-43-2725,CC_residual_temperature_correction_factor_b,1.16E-04,
-43-2725,CC_residual_temperature_correction_factor_c,-9.01E-07,
-43-2725,CC_residual_temperature_correction_factor_e,3.60E-02,
+43-2725,CC_frequency_offset,-8.164389e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2725_Calibration_2016-11-24.zip > SBE 43F O2725 SOCAdjusted.pdf and 2725.cal
+43-2725,CC_oxygen_signal_slope,2.9433e-004,adjusted Soc (O2 signal slope) value
+43-2725,CC_residual_temperature_correction_factor_a,-2.384006e-003,
+43-2725,CC_residual_temperature_correction_factor_b,1.155854e-004,
+43-2725,CC_residual_temperature_correction_factor_c,-9.012087e-007,
+43-2725,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20161124.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20171223.csv [hunk 11]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2725,CC_frequency_offset,-7.845930e+002,
-43-2725,CC_oxygen_signal_slope,2.938453e-004,
+43-2725,CC_frequency_offset,-7.845930e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2725_Calibration_2017-12-23.zip > SBE 43F O2725 SOCAdjusted.pdf and 2725.cal
+43-2725,CC_oxygen_signal_slope,2.9669e-004,adjusted Soc (O2 signal slope) value
 43-2725,CC_residual_temperature_correction_factor_a,-3.734256e-003,
 43-2725,CC_residual_temperature_correction_factor_b,1.585968e-004,
 43-2725,CC_residual_temperature_correction_factor_c,-2.222710e-006,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20171223.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20150123.csv [hunk 12]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2726,CC_frequency_offset,-801.3801,
-43-2726,CC_oxygen_signal_slope,0.0002893416,
+43-2726,CC_frequency_offset,-801.3801,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2726_Calibration_2015-01-23.zip > SBE 43F O2726 SOCAdjusted.pdf and 2726.cal
+43-2726,CC_oxygen_signal_slope,3.0073e-004,adjusted Soc (O2 signal slope) value
 43-2726,CC_residual_temperature_correction_factor_a,-0.004030344,
 43-2726,CC_residual_temperature_correction_factor_b,0.0001736643,
 43-2726,CC_residual_temperature_correction_factor_c,-2.460596e-06,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20150123.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20151224.csv [hunk 13]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2726,CC_frequency_offset,-8.012258e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2726_Calibration_2015-12-24.zip > SBE 43F O2726 SOCAdjusted.pdf and 2726.cal
+43-2726,CC_oxygen_signal_slope,3.0809e-004,adjusted Soc (O2 signal slope) value
+43-2726,CC_residual_temperature_correction_factor_a,-3.264675e-003,
+43-2726,CC_residual_temperature_correction_factor_b,1.871381e-004,
+43-2726,CC_residual_temperature_correction_factor_c,-3.154411e-006,
+43-2726,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20151224.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20161213.csv [hunk 14]
```diff
@@ -1,7 +1,7 @@
...
-43-2726,CC_residual_temperature_correction_factor_a,-3.39E-03,
-43-2726,CC_residual_temperature_correction_factor_b,1.49E-04,
-43-2726,CC_residual_temperature_correction_factor_c,-2.34E-06,
-43-2726,CC_residual_temperature_correction_factor_e,3.60E-02,
+43-2726,CC_frequency_offset,-7.987092e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2726_Calibration_2016-12-13.zip > SBE 43F O2726 SOCAdjusted.pdf and 2726.cal
+43-2726,CC_oxygen_signal_slope,3.0945e-004,adjusted Soc (O2 signal slope) value
+43-2726,CC_residual_temperature_correction_factor_a,-3.385586e-003,
+43-2726,CC_residual_temperature_correction_factor_b,1.489891e-004,
+43-2726,CC_residual_temperature_correction_factor_c,-2.340215e-006,
+43-2726,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20161213.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20140828.csv [hunk 15]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-784.19,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2978_Calibration_2014-08-28.zip > 43F O2978 SOCAdjusted 28Aug14.pdf
+43-2978,CC_oxygen_signal_slope,2.4685e-004,adjusted Soc (O2 signal slope) value
+43-2978,CC_residual_temperature_correction_factor_a,-4.0705e-003,
+43-2978,CC_residual_temperature_correction_factor_b,2.3506e-004,
+43-2978,CC_residual_temperature_correction_factor_c,-3.8155e-006,
+43-2978,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20140828.csv]: Looks good. No .cal file in source file

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20150710.csv [hunk 16]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-7.920499e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2978_Calibration_2015-07-10.zip > SBE 43F O2978 SOCAdjusted.pdf and 2978.cal
+43-2978,CC_oxygen_signal_slope,2.5786e-004,adjusted Soc (O2 signal slope) value
+43-2978,CC_residual_temperature_correction_factor_a,-4.098681e-003,
+43-2978,CC_residual_temperature_correction_factor_b,2.579854e-004,
+43-2978,CC_residual_temperature_correction_factor_c,-3.715804e-006,
+43-2978,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20150710.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20170708.csv [hunk 17]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-793.0019,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2978_Calibration_2017-07-08.zip > SBE 43F O2978 SOCAdjusted.pdf and 2978.cal
+43-2978,CC_oxygen_signal_slope,2.6602e-004,adjusted Soc (O2 signal slope) value
+43-2978,CC_residual_temperature_correction_factor_a,-4.105169e-003,
+43-2978,CC_residual_temperature_correction_factor_b,2.628279e-004,
+43-2978,CC_residual_temperature_correction_factor_c,-3.331433e-006,
+43-2978,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20170708.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20180613.csv [hunk 18]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-2978,CC_frequency_offset,-7.908247e+002,
-43-2978,CC_oxygen_signal_slope,2.319300e-004,
+43-2978,CC_frequency_offset,-7.908247e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2978_Calibration_2018-06-13.zip > SBE 43F O2978 SOCAdjusted  13Jun18.pdf and SBE43F_2978.cal
+43-2978,CC_oxygen_signal_slope,2.4210e-004,adjusted Soc (O2 signal slope) value
 43-2978,CC_residual_temperature_correction_factor_a,-5.183874e-003,
 43-2978,CC_residual_temperature_correction_factor_b,2.601490e-004,
 43-2978,CC_residual_temperature_correction_factor_c,-3.754855e-006,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20180613.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20170725.csv [hunk 19]
```diff
@@ -1,7 +1,7 @@
...
-43-3370,CC_oxygen_signal_slope,0.000284625,
-43-3370,CC_residual_temperature_correction_factor_a,-0.004185138,
-43-3370,CC_residual_temperature_correction_factor_b,0.000189943,
-43-3370,CC_residual_temperature_correction_factor_c,-2.40E-06,
+43-3370,CC_frequency_offset,-851.4512,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3370_Calibration_2017-07-25.zip > SBE 43F O3370  SOCAdjusted 25Jul17.pdf and 3370.cal
+43-3370,CC_oxygen_signal_slope,2.9536e-004,adjusted Soc (O2 signal slope) value
+43-3370,CC_residual_temperature_correction_factor_a,-4.185138e-003,
+43-3370,CC_residual_temperature_correction_factor_b,1.899428e-004,
+43-3370,CC_residual_temperature_correction_factor_c,-2.398023e-006,
 43-3370,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20170725.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20180526.csv [hunk 20]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-3370,CC_frequency_offset,-8.425638e+002,
-43-3370,CC_oxygen_signal_slope,2.881344e-004,
+43-3370,CC_frequency_offset,-8.425638e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3370_Calibration_2018-05-26.zip > SBE 43F O3370 SOCAdjusted  26May18.pdf and SBE43F_3370.cal
+43-3370,CC_oxygen_signal_slope,2.9683e-004,adjusted Soc (O2 signal slope) value
 43-3370,CC_residual_temperature_correction_factor_a,-3.802380e-003,
 43-3370,CC_residual_temperature_correction_factor_b,2.056000e-004,
 43-3370,CC_residual_temperature_correction_factor_c,-2.979834e-006,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20180526.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03506__20170314.csv [hunk 21]
```diff
@@ -1,7 +1,7 @@
...
-43-3506,CC_residual_temperature_correction_factor_a,-3.59E-03,
-43-3506,CC_residual_temperature_correction_factor_b,2.00E-04,
-43-3506,CC_residual_temperature_correction_factor_c,-3.37E-06,
-43-3506,CC_residual_temperature_correction_factor_e,3.60E-02,
+43-3506,CC_frequency_offset,-8.666162e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3506_Calibration_2017-03-14.zip > 43-3506 Oxygen Cal.pdf and 43-3506.cal
+43-3506,CC_oxygen_signal_slope,3.0322e-004,adjusted Soc (O2 signal slope) value
+43-3506,CC_residual_temperature_correction_factor_a,-3.594980e-003,
+43-3506,CC_residual_temperature_correction_factor_b,2.003504e-004,
+43-3506,CC_residual_temperature_correction_factor_c,-3.374411e-006,
+43-3506,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03506__20170314.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03506__20180526.csv [hunk 22]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-3506,CC_frequency_offset,-8.740126e+002,
-43-3506,CC_oxygen_signal_slope,2.980255e-004,
+43-3506,CC_frequency_offset,-8.740126e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3506_Calibration_2018-05-26.zip > SBE 43F O3506 SOCAdjusted 26May18.pdf and SBE43F_3506.cal
+43-3506,CC_oxygen_signal_slope,3.0960e-004,adjusted Soc (O2 signal slope) value
 43-3506,CC_residual_temperature_correction_factor_a,-3.438634e-003,
 43-3506,CC_residual_temperature_correction_factor_b,1.922919e-004,
 43-3506,CC_residual_temperature_correction_factor_c,-3.302145e-006,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03506__20180526.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03523__20170314.csv [hunk 23]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3523,CC_frequency_offset,-8.221968e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3523_Calibration_2017-03-14.zip > 43-3523 Oxygen Cal.pdf and 43-3523.cal
+43-3523,CC_oxygen_signal_slope,3.4103e-004,adjusted Soc (O2 signal slope) value
+43-3523,CC_residual_temperature_correction_factor_a,-3.531825e-003,
+43-3523,CC_residual_temperature_correction_factor_b,1.687575e-004,
+43-3523,CC_residual_temperature_correction_factor_c,-3.091832e-006,
+43-3523,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03523__20170314.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-03523__20180525.csv [hunk 24]
```diff
@@ -1,6 +1,6 @@
 serial,name,value,notes
-43-3523,CC_frequency_offset,-8.250858e+002,
-43-3523,CC_oxygen_signal_slope,3.365236e-004,
+43-3523,CC_frequency_offset,-8.250858e+002,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_3523_Calibration_2018-05-25.zip > SBE 43F O3523 SOCAdjusted 25May18.pdf and SBE43F_3523.cal
+43-3523,CC_oxygen_signal_slope,3.5762e-004,adjusted Soc (O2 signal slope) value
 43-3523,CC_residual_temperature_correction_factor_a,-3.430888e-003,
 43-3523,CC_residual_temperature_correction_factor_b,1.642034e-004,
 43-3523,CC_residual_temperature_correction_factor_c,-2.967717e-006,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-03523__20180525.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20110702.csv [hunk 25]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-0209,CC_frequency_offset,-8.057700e+02,source_file DOFST-K_SBE43F_SN_0209_Calibration_2011-07-02.txt
+43-0209,CC_oxygen_signal_slope,3.226900e-04,adjusted Soc not available fr vendor
+43-0209,CC_residual_temperature_correction_factor_a,-3.124700e-03,
+43-0209,CC_residual_temperature_correction_factor_b,1.602100e-04,
+43-0209,CC_residual_temperature_correction_factor_c,-2.618200e-06,
+43-0209,CC_residual_temperature_correction_factor_e,3.600000e-02,
```
Comment (reedan88) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20110702.csv]: Looks good.

File: calibration/DOFSTK/CGINS-DOFSTK-02628__20161221.csv [hunk 26]
```diff
@@ -1,7 +1,7 @@
 serial,name,value,notes
-43-2628,CC_frequency_offset,828.6498,
-43-2628,CC_oxygen_signal_slope,0.000282756,
-43-2628,CC_residual_temperature_correction_factor_a,-2.50E-03,
-43-2628,CC_residual_temperature_correction_factor_b,2.06E-04,
-43-2628,CC_residual_temperature_correction_factor_c,-2.61E-06,
-43-2628,CC_residual_temperature_correction_factor_e,3.60E-02,
+43-2628,CC_frequency_offset,828.6498,date in filename comes from Soc-adjusted caldate; source_file DOFST-K_SBE43F_SN_2628_Calibration_2016-12-21.zip > SBE 43F O2628 SOCAdjusted 21Dec16.pdf and 2628.cal
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/DOFSTK/CGINS-DOFSTK-02628__20161221.csv]: good catch - fixed!

File: calibration/DOFSTK/CGINS-DOFSTK-03523__20180525.csv (hunk 24 above)
Comment (johnmlund) [reply calibration/DOFSTK/CGINS-DOFSTK-03523__20180525.csv]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02628__20161221.csv (hunk 26 above)
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/DOFSTK/CGINS-DOFSTK-02628__20161221.csv]: Looks good now!

**Pull request merged**